### PR TITLE
animation

### PR DESCRIPTION
**Pull request by joonhocho**

### CODE REVIEW

File: packrat/styles/keeper/tagbox.css [hunk 1]
```diff
@@ -314,6 +316,8 @@
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
+  -moz-transition: -moz-transform 0.3s ease;
+  -webkit-transition: -webkit-transform 0.3s ease;
```
Comment (2is10) [inline packrat/styles/keeper/tagbox.css:320]: `-moz-transition`, `-moz-transform`, and `-webkit-transition` are all deprecated in the last few Chrome/Firefox releases, so we just use this pattern in the extension code:

```
transition: -webkit-transform .3s ease;
transition: transform .3s ease;
```

File: packrat/styles/keeper/tagbox.css [hunk 2]
```diff
@@ -322,6 +326,8 @@
   width: 18px;
 }
 .kifi-tagbox-tag.busy {
```
Comment (2is10) [inline packrat/styles/keeper/tagbox.css:328]: `.kifi-busy`

File: packrat/styles/keeper/tagbox.css (hunk 1 above)
Comment (joonhocho) [inline packrat/styles/keeper/tagbox.css:320]: That was originally my thought, then it wasn't working on my browser. So, I changed it to vendor prefixes.
Weird.

Comment (2is10) [inline packrat/styles/keeper/tagbox.css:320]: Interesting. We should try to get to the bottom of it.

File: packrat/styles/keeper/tagbox.css (hunk 2 above)
Comment (joonhocho) [inline packrat/styles/keeper/tagbox.css:328]: will be fixed

**Pull request merged**